src/sdrf_pipelines/converters/openms/openms.py

```
@@ -0,0 +1,540 @@
+            f2c.file2pctolunit[raw] = unit
+        else:
+            warning_message = "No precursor mass tolerance set. Assuming 10 ppm."
+            self.warnings[warning_message] = self.warnings.get(warning_message, 0) + 1
+            f2c.file2pctol[raw] = "10"
+            f2c.file2pctolunit[raw] = "ppm"
+
+        if "comment[fragment mass tolerance]" in row:
+            f_tol_str = row["comment[fragment mass tolerance]"].strip()
+            tol, unit = parse_tolerance(f_tol_str)
+            if tol is None or unit is None:
+                raise ValueError(f"Cannot read precursor mass tolerance: {f_tol_str}")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)